routes/base_node.js

```diff
@@ -13,12 +13,18 @@ const {
 } = require('../models/base_node')
 const { baseNode } = require('../protos')
 
+function errorObject (name, message) {
+  return { error: { name, message } }
+}
+
+const internalError = errorObject('internal', 'Internal error')
```

Comment:
This would be better as a function. I thought `internalError` was a copy paste error for a missing local variable, not realising it was a global

Reply:
Perhaps:

```
function sendDefaultError(res) {
   const internalError = errorObject('internal', 'Internal error')
   return res.status(500).json(internalError);
}
```

Reply:
Perhaps it needs to be named more like a 'static' constant i.e `INTERNAL_ERROR`?

Reply:
Agreed - Updated!